corehq/ex-submodules/pillowtop/processors/form.py

```diff
@@ -91,7 +85,7 @@ def mark_latest_submission(domain, user_id, app_id, build_id, version, metadata,
         metadata
     )
 
-    if last_submission is None or last_submission.submission_date < received_on_datetime:
+    if last_submission is None or last_submission.submission_date <= received_on_datetime:
```

Comment:
what also might be nice is if we had a `debounce` method (http://underscorejs.org/#debounce). only update the user to the last form submission after we haven't seen a form submission for an hour or so. admittedly seems somewhat hard to implement. one idea:

1. use celery `eta` param to update the user in 1 hour.
2. save task id to redis
3. if another form comes in, and we have a task waiting to be processed (fetched from redis), revoke and reschedule for another hour